core/src/testUtils/groovy/nu/yona/server/test/Buddy.groovy

```diff
@@ -0,0 +1,25 @@
+package nu.yona.server.test
+
+import nu.yona.server.YonaServer
+import groovy.json.*
+
+class Buddy
+{
+	final String nickname
+	final String receivingStatus
+	final String sendingStatus
+	final User user;
+	final String url
+	Buddy(def json)
+	{
+		this.nickname = json.nickname
+		this.receivingStatus = json.receivingStatus
+		this.sendingStatus = json.sendingStatus
+		// TODO: Is it appropriate to return the user with all null data? I'd rather omit the user in that case.
```

Comment:
It's better to create Tasks instead of TODO comments.
For this case we maybe should return a user with most `null` data and `firstName` equal to `"(removed)"?